src/components/top-bar.js

```diff
@@ -40,28 +32,16 @@ const useStyles = makeStyles(theme => ({
     }
 }));
 
-const TopBar = () => {
-
-    const dispatch = useDispatch();
-
-    const dark = useSelector(state => state.darkTheme);
-
-    const useName = useSelector(state => state.useName);
-
-    const intl = useIntl();
+const TopBar = (props) => {
 
     const classes = useStyles();
 
-    function switchTheme() {
-        dispatch(selectDarkTheme(!dark));
-    }
-
-    const handleToggleUseName = () => {
-        dispatch(toggleUseNameState());
+    const onParametersClick = () => {
+      if (props.onParametersClick) {
+          props.onParametersClick();
+      }
     };
```

Comment:
When you click the button several times, it opens several windows on top of each other. Would it be possible to close on the second click the parameter window instead? (so same effect as the close button)

Reply:
Thanks for the catch, it is a bug.
Close the parameters pane or disable the button in the top bar ?

Reply:
Ok for closing the parameters pane.